### PR TITLE
scipoptsuite 6.0.0 (new formula)

### PR DESCRIPTION
**Pull request by Zialus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by DomT4 — DISMISSED** [review]

File: Formula/scipoptsuite.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/scip", "--version"
```
[the remark this answers is not in the document]
Comment (Zialus) [reply Formula/scipoptsuite.rb]: I'm adding the test at the end of this page http://scip.zib.de/doc/html/index.php#QUICKSTART

File: Formula/scipoptsuite.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  sha256 "a91119687e521575070c43eaa30bc15ce461e9dc414651793ea4c4ca33b89750"
+
+  depends_on "cmake" => :build
+  depends_on "gmp"
+  depends_on "readline"
+
+  def install
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
```
[the remark this answers is not in the document]
Comment (Zialus) [reply Formula/scipoptsuite.rb]: I've just found out that `-DSHARED=on` is not even needed anymore

**Comment by DomT4** [comment]
I'm not actually 100% convinced we're allowed to legally distribute this.

> You are allowed to retrieve a copy of SCIP for research purposes as a member of a noncommercial and academic institution. In order to get a copy of SCIP, you need to certify that you are a member of a noncommercial, academic institution and accept the ZIB Academic License.

Examining the actual license:
> 1. This license applies to you only if you are a member of a noncommercial 
   and academic institution, e.g., a university. The license expires as
   soon as you are no longer a member of this institution. 

That's specific enough to exclude package managers, kinda leaving the question... well, what license does apply to us?

Beyond that there are obvious impracticalities around section 3c:

```
3. You may copy and distribute the Program or work based on the Program 
   in source, object, or executable form provided that you also meet
   all of the following conditions: 
[snip]
   c. You must keep track of access to the Program (e.g., similar to the 
      registration procedure at ZIB).
```

IANAL, but I'm pretty reluctant to package this based on that license without some clarification from either upstream.

**Comment by DomT4** [comment]
Yeah, we're going to close this out on the above basis of believing this software may be legally problematic for Homebrew to distribute. We would be willing to reconsider this decision if the license was changed to a recognised open-source license.

In the circumstances the license seems to be trying to achieve it was suggested upstream may wish to explore GPLv3 with the option of commercial licensing.

**Pull request closed without merging**